### ISSUE TITLE
Backup command is not quite idempotent?

### ISSUE THREAD

**Issue by jimbojsb** [lra/mackup#1504]
Running `mackup backup` a second time (using icloud as the storage engine) requires me to either pipe yes to the command or manually confirm every folder over again. This is annoying when adding a new application to the backup. Am I doing it wrong?

Example:

```[josh@tangerine ~]$ mackup backup
Backing up Library/Application Support/Adobe/CameraRaw ...
A folder named /Users/josh/Library/Mobile Documents/com~apple~CloudDocs/Mackup/Library/Application Support/Adobe/CameraRaw already exists in the backup.
Are you sure that you want to replace it ? <Yes|No>yes
```

**Comment by jamiew** [comment]
I've noticed this occurs only when an app is currently running, e.g. I see this every time I run `backup` with Fantastical and Magnet. Quitting those apps and running again works without issue.

**Comment by lra** [comment]
Yes. This is quite annoying and IMO, files causing this should be removed from syncing until Mackup supports copying.

Feel free to PR to remove any file causing this behavior.
